README.md

```diff
@@ -1,2 +1,199 @@
-# Fluent
-A brand new framework to build amazing android applications
+
+# Fluent 
+
+Fluent is a lightweight framework thats helps you to create an entirely Android Application following SOLID pattern with an unidirectional flow reactive or not.
+
+The main concepts in this architecture are : `State`, `View`, `Store`, `Job` and `Hub` 
+
+---
+>![fluent diagram](https://image.ibb.co/cAXPhH/Screen_Shot_2018_04_11_at_12_09_41.png)
+---
+
+## State
+
+We recommend to define one `State` for each one of your screens.
+
+It holds just `type() : StateType` and should be inherit to fit your needs. As the example above:
+
+```kotlin
+data class LoginState(val type: StateType = StateType.Initial) : State {
```

Comment:
This example isn't good at all. We should show a state implementation with more view related attributes.